nimble/controller/src/ble_ll_scan.c

```diff
@@ -2103,6 +2121,20 @@ ble_ll_scan_get_addr_from_ext_adv(uint8_t *rxbuf, struct ble_mbuf_hdr *ble_hdr,
 
     return 0;
 }
+
+static int
+ble_ll_scan_get_adi(struct ble_ll_aux_data *aux_data)
```

Comment:
I'd make adi optional output parameter instead of combining it in return code

Reply:
agreed